scripts/test-file-size.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+# add any files you want the test to ignore between the two ENDOFLIST placeholders. 
```

Comment:
Maybe document that you have to assume that you are in the root of the repo when specifying the path? Is the `./` needed?

Reply:
Do you mean, if you are running the tests locally?

Reply:
Or do you mean to ensure paths are specified correctly in the script if it's edited?